Doctrine/Orm/Filter/DateFilter.php

```diff
@@ -175,7 +175,11 @@ public function getDescription(ResourceInterface $resource)
         }
 
         foreach ($properties as $property => $nullManagement) {
-            if (!$this->isPropertyMapped($property, $resource)) {
+            if (
+                !$this->isPropertyEnabled($property)
+                || !$this->isPropertyMapped($property, $resource)
```

Comment:
CS inconsistency. Do we have the `||` operator at the end of the previous line as above, or at the beginning of the line?